lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/VortexLauncher.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.vortex;
+
+import org.apache.reef.client.DriverLauncher;
+import org.apache.reef.client.LauncherStatus;
+import org.apache.reef.runtime.local.client.LocalRuntimeConfiguration;
+import org.apache.reef.runtime.mesos.client.MesosClientConfiguration;
+import org.apache.reef.runtime.yarn.client.YarnClientConfiguration;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.exceptions.InjectionException;
+import org.apache.reef.vortex.api.VortexUserCode;
+
+/**
+ * Launches a Vortex Job.
+ */
+public final class VortexLauncher {
+  private VortexLauncher() {
+  }
+
+  private static final int MAX_NUMBER_OF_EVALUATORS = 10;
+
+  public static LauncherStatus launchLocal(final String jobName,
+                                           final Class<? extends VortexUserCode> vortexUserCode,
+                                           final int numOfExecutors,
+                                           final int executorMemory,
+                                           final int executorCores) {
+    final Configuration runtimeConfiguration = LocalRuntimeConfiguration.CONF
```

Comment:
Inconsistent naming: `runtimeConfiguration` and `vortexConf`
`runtimeConfiguration` and `vortexConfiguration` OR
`runtimeConf` and `vortexConf`